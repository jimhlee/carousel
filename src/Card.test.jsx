import { it, expect } from "vitest";

import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

 const TEST_IMAGE =   {
    src: "test1.com",
    caption: "testing image 1"
  }


it('renders without crashing', function() {
    render(<Card
        caption="testing image 1"
        src="test1.com"
        currNum="1"
        totalNum="2"
        />
    );
})

it('matches snapshot', function() {
    const { container } = render(
        <Card
        caption="testing image 1"
        src="test1.com"
        currNum="1"
        totalNum="2"
        />
      );
    expect(container).toMatchSnapshot();
})