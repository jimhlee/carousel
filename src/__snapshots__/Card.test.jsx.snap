// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testing image 1
    </h4>
    <img
      alt="testing image 1"
      class="Card-image"
      src="test1.com"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      2
      .
    </small>
  </div>
</div>
`;
